packages/sdks/output/qwik/src/blocks/button/button.tsx

```diff
@@ -0,0 +1,202 @@
+// GENERATED BY MITOSIS
+
+import { component$, useStore, h, Fragment } from "@builder.io/qwik";
+export const COMPONENT = {
```

Comment:
What are these for? Usually this should compile away

Reply:
I keep them here for debugging purposes. They will be removed.